openshift/worker.yaml

```diff
@@ -14,6 +14,25 @@ labels:
   app: masu
   template: masu-template
 objects:
+- kind: Service
```

Comment:
Might be nicer to pull this out into its own yaml if that we could replace the `${NAME}-worker-0` if we scale up, unless there is another way to dynamically create the service.